File: Task 6/6b.py
'''6b.)Write a Python program to read and display the content of a given CSV file'''
import pandas as pd
'''names = pd.read_csv("C:\\21241A6625\\Task 6\\names.csv")
names = pd.read_csv("C:/21241A6625/Task 6/names.csv")
names = pd.read_csv(r"C:\21241A6625\Task 6\names.csv")
'''
names = pd.read_csv(input()+".csv")
print("Contents of file:")
print(names.shape)
print (names.head())
x = names['Age']
y = names.drop('Age',axis=1)
print('AGE:')
print (x.head())
print('Contents after dropping Age:')
print (y.head())
print("Contents of last two members of file:")
print(y.tail(2))

File: R-Programming/Task 6/6b.R
#TASK 6
#(b) Write a R program for multiplication of two matrices.
a = matrix(c(1,3,5,7), ncol=2, nrow=2)
b = matrix(c(2,4,6,8), ncol=2, nrow=2)
a*b
a %*% b
#Output:
#     [,1] [,2]
#[1,]    2   30
#[2,]   12   56
#     [,1] [,2]
#[1,]   22   46
#[2,]   34   74
